src/justenoughicu.c

```diff
@@ -28,24 +29,32 @@ int icu_case(lua_State *L) {
   u_strFromUTF8(input_as_uchar, l, &l, input, input_l, &err);
 
   /* Now do the conversion */
-  conversion_function_t conversion;
   UChar *output;
   int32_t l2 = 0;
 
-  if (strcmp(recase, "upper") == 0) {
-    conversion = u_strToUpper;
-  } else if (strcmp(recase, "lower") == 0) {
-    conversion = u_strToLower;
-  // } else if (strcmp(recase, "title") == 0) {
-    // conversion = u_strToTitle;
+  if (strcmp(recase, "title") == 0) {
+    conversion_function_ti conversion;
+    UBreakIterator *iter;
```

Comment:
I've seen odder ways — I saw my own earlier attempts at this before I sorted out even to this point! :wink:

Thanks for the feedback. I just pushed another commit that takes your suggestions into account as best I understood them.